src/base1/patternfly-overrides.less

```diff
@@ -1,4 +1,9 @@
 // Global Cockpit overrides for PatternFly variables
+// PF4 React components need this css to look as expected
+// See https://github.com/patternfly/patternfly-react/tree/master/packages/patternfly-4/react-core#pre-requisites
+// This needs to be imported before anything else because it's resetting stuff which later we or other PF4 css rules
+// will set
+@import '../../node_modules/@patternfly/react-core/dist/styles/base.css';
```

Comment:
This should probably not be imported here - we need to find a place for that.
CC @garrett